users-guide/index.qmd

```diff
@@ -0,0 +1,13 @@
+---
+title: Use's Guide
```

Comment:
Users? or Usage?

Reply:
Actually, I think it should be [User Guide](https://en.wikipedia.org/wiki/User_guide). Renamed the folder and fixed titles accordingly.